tests/stress/stress_test.go

```diff
@@ -15,9 +16,9 @@ const (
 )
 
 type StressTestCase struct {
```

Comment:
Overall, these changes are looking really good :+1: 

I think we're overthinking the Deploy for Task vs. Service. 
There's really two parts I think we need to tackle: the dependency and the command. 


Since we already have an environment dependency for all of our entities (e.g. you could never run with `NumServices > 0 && NumEnvironments == 0`), I think it's reasonable and consistent to require that `NumDeploys > 0` for any tests with Tasks and/or Services.

As for the command in the Deploy, I don't think it actually matters. We don't need tasks to exit, I think it's not unreasonable that we just hardcode the command into `Dockerrun.aws.json`. This will simplify the test code and the terraform config a bit (since we won't need to template the deploy).

Reply:
Added a couple of top-of-test checks for Environments and Deploys given Services and/or Tasks. Also reverted hardcoded deploy command.

✅